content/docs/0.7.x/operate/install/index.md

```diff
@@ -219,288 +227,224 @@ Every Keptn release provides binaries for the Keptn CLI. These binaries are avai
-<p>
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
 
-* If you are using **ClusterIP** in combination with *port-forward* to access Keptn, the Keptn endpoint is on `localhost` and the `port` you forwarded Keptn to. For example, the Keptn API endpoint is then `http://localhost:8080/api`
+      ```console
+    kubectl -n ingress-nginx svc ingress-nginx
+      ```
 
-</p>
-</details>
+    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown next, `nip.io` is used as wildcard DNS for the IP address.)
+
+      ```yaml
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+      annotations:
+        kubernetes.io/ingress.class: nginx
+      name: api-keptn-ingress
+      namespace: keptn
+    spec:
+      rules:
+      - host: <IP-ADDRESS>.nip.io
+        http:
+          paths:
+          - backend:
+              serviceName: api-gateway-nginx
+              servicePort: 80
+      ```
 
-<details><summary>Keptn API endpoint when exposing via **ClusterIP** in combination with an *Ingress Controller*</summary>
-<p>
+      ```console
+    kubectl apply -f ingress-manifest.yaml
+      ```
 
-* If you are using **ClusterIP** in combination with an *Ingress Controller* to expose Keptn, get the HOST of the `api-keptn-ingress`using the next command. Consequently, the Keptn API endpoint is: `http://<HOST>/api`
+    </p>
+    </details>
 
-  ```console
-kubectl -n keptn get ingress api-keptn-ingress
-  ```
+1. **Get Keptn endpoint:** Get the HOST of the `api-keptn-ingress` using the command below. The Keptn API endpoint is: `http://<HOST>/api`
 
-  ```console
-NAME                HOSTS                  ADDRESS         PORTS   AGE
-api-keptn-ingress   <HOST>                 x.x.x.x   80      48m
-  ```
+    ```console
+  kubectl -n keptn get ingress api-keptn-ingress
+    ```
 
-</p>
-</details>
+    ```console
+  NAME                HOSTS                  ADDRESS         PORTS   AGE
+  api-keptn-ingress   <HOST>                 x.x.x.x   80      48m
+    ```
 
+    Optional: Store Keptn API endpoint in an environment variable:
+    ```console
+    KEPTN_API_ENDPOINT=http://<HOST>/api
+    ```
 
-<details><summary>Keptn API endpoint when exposing via **LoadBalancer**</summary>
-<p>
+:warning: **Warning:** If you do not setup TLS encryption, all your traffic to and from the Keptn endpoint is not encrypted.
 
-* If you are using a **LoadBalancer** to expose Keptn, get the EXTERNAL-IP of the `api-gateway-ngix` using the next command. Consequently, the Keptn API endpoint is: `http://<ENDPOINT_OF_API_GATEWAY>/api`
+### Option 4: Access Keptn via a Port-forward
+This option exposes Keptn on a *cluster-internal* IP .
```

Comment:
Maybe we could write the sentence like that: _This option does not expose Keptn, but rather allows accessing Keptn on its *cluster-interanl* IP._

To make it clear that this is not exposing Keptn.